pcdsdevices/jet.py

```diff
@@ -74,19 +74,19 @@ class BeckhoffJetManipulator(BaseInterface, GroupDevice):
 
     tab_component_names = True
 
-    x = Cpt(BeckhoffAxis, ':X', kind='normal')
-    y = Cpt(BeckhoffAxis, ':Y', kind='normal')
-    z = Cpt(BeckhoffAxis, ':Z', kind='normal')
+    x = Cpt(OldBeckhoffAxis, ':X', kind='normal')
```

Comment:
Would recommend using the versioned name here since it's in the PR.
Is the `Old` alias even used outside of this?

Reply:
I think it might be used in a beamline config somewhere. I left it in out of an abundance of caution.